doc/man3/BN_rand.pod

```diff
@@ -37,7 +43,14 @@ If B<bits> is 1 then B<top> cannot also be B<BN_RAND_FLG_TOPTWO>.
 BN_rand_range() generates a cryptographically strong pseudo-random
 number B<rnd> in the range 0 E<lt>= B<rnd> E<lt> B<range>.
 
-The PRNG must be seeded prior to calling BN_rand() or BN_rand_range().
+BN_priv_rand() and BN_priv_rand_range() have the same semantics as
+BN_rand() and BN_rand_range() respectively.  They are intended to be
+used for generating values that should remain private, and mirror the
+same difference between RAND_bytes() and RAND_priv_bytes().
+
+An error occurs in the underlying B<RAND> functions if the CSPRNG has
+not been seeded with enough randomness to ensure an unpredictable byte
+sequence.
```

Comment:
I mirrored the note contained in RAND_bytes.3 for
> An error occurs in the underlying B<RAND> functions if the CSPRNG has not been seeded with enough randomness to ensure an unpredictable byte sequence.

Reading RAND.7, to me it seems that this, and the original note, are in conflict to the following statement:

> The default random generator will initialize automatically on first use and will be fully functional without having to be initialized ('seeded') explicitly.  It seeds and reseeds itself automatically using trusted random sources provided by the operating system.

- Should I remove the above note?
- What about the one in RAND_bytes.3?

Reply:
I guess it's pretty much done now, when I opened it it was limited only to `BN_priv_rand()` and `RAND_priv_bytes()`, it was only after the feedback on `RAND(7)` and `RAND_DRBG(7)` that I noticed that also `RAND(7)` needed updating, because it had the same notice note on "long-term secrets" rather than "any private value".

Commit d246d50 too is just an improvement on some feedback I received in this PR, about including a note about errors when the CSPRNG does not have enough entropy: upon reading `RAND(7)` I initially misinterpreted the intention of the note when compared with the note in `RAND(7)` about automatic initialization and reseeding: when I (5 minutes later) realized that the note on `RAND_bytes(3)` was to remind API users to always check for the error return value, I made that more explicit so that other silly users like me don't fall in the same trap!